src/Symfony/Bridge/Monolog/Formatter/ConsoleFormatter.php

```diff
@@ -11,45 +11,191 @@
 
 namespace Symfony\Bridge\Monolog\Formatter;
 
-use Monolog\Formatter\LineFormatter;
-use Monolog\Logger;
+use Monolog\Formatter\FormatterInterface;
+use Symfony\Component\VarDumper\Cloner\Data;
+use Symfony\Component\VarDumper\Cloner\Stub;
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
 
 /**
  * Formats incoming records for console output by coloring them depending on log level.
  *
  * @author Tobias Schultze <http://tobion.de>
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
  */
-class ConsoleFormatter extends LineFormatter
+class ConsoleFormatter implements FormatterInterface
 {
-    const SIMPLE_FORMAT = "%start_tag%[%datetime%] %channel%.%level_name%:%end_tag% %message% %context% %extra%\n";
+    const SIMPLE_FORMAT = "%datetime% %start_tag%%level_name%%end_tag% <comment>[%channel%]</comment> %message%%context%%extra%\n";
+    const SIMPLE_DATE = 'H:i:s';
+
+    private static $levelColorMap = array(
+        100 => 'fg=white',
+        200 => 'fg=green',
+        250 => 'fg=blue',
+        300 => 'fg=cyan',
+        400 => 'fg=yellow',
+        500 => 'fg=red',
+        550 => 'fg=red',
+        600 => 'fg=white;bg=red',
+    );
+
+    private $options;
+    private $cloner;
+    private $outputBuffer;
+    private $dumper;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct($options = array())
+    {
+        // BC Layer, old constructor:
```

Comment:
Actually, it's "documented" just bellow in the trigger error message.

For the record, I added what you asked for, then I added the deprecation, then I removed because it was a kind of duplicate. and now I notice I forgot to remove `, old constructor:`